apps/src/sites/code.org/pages/views/workshop_search.js

```diff
@@ -10,30 +11,265 @@ const markerCustererOptions = {
   gridSize: 10
 };
 
-var gmap,
+var map,
   markersByLocation = {},
   infoWindow,
   markerClusterer;
 
 $(document).ready(function() {
-  initializeMap();
-  loadWorkshops();
+  if (window.location.search.includes('mapbox')) {
+    $('#gmap').hide();
+    $('#mapbox-map').show();
+    $('#geocomplete').hide();
+    $('#geocomplete-form').hide();
+    $('#mapboxgeocoder').show();
+    initializeMapboxMap();
+    map.on('load', loadMapboxWorkshops);
+  } else {
+    initializeGoogleMap();
+    loadGoogleMapWorkshops();
+  }
 });
 
-function initializeMap() {
+function initializeMapboxMap() {
+  map = new mapboxgl.Map({
+    container: 'mapbox-map',
+    style: 'mapbox://styles/mapbox/streets-v11',
+    center: [-95.665, 37.6],
+    zoom: 3
+  });
+
+  map.dragRotate.disable();
+  map.scrollZoom.disable();
+  map.dragPan.disable();
```

Comment:
I guess the density & distribution of workshops make it okay not disable movement & zooming?